### PR TITLE
Changing MapBox configuration

### LINKED ISSUE #463: [web-ui] There are strange objects
Firefox
---------
kytos_firefox

Google Chrome
---------------------
kytos_google_chrome

Description
---------------
Some objects related with MapBox appear in google-chrome and don't appear in Firefox.

### PR DESCRIPTION
**Pull request by macartur**
- Removing attributionControl and zoomControl from MapBox
- Changing MapBox styles
- Fix #463

### CODE REVIEW

**Comment by diraol** [comment]
@macartur can you please add a screenshot showing where the mapbox/leaflet logo will be placed? [add it on a comment on this PR]
Thanks!

**Comment by macartur** [comment]
OK. I took some screenshots using firefox and google-chrome.

Firefox (54.0, 64bits)
================


firefox-printscreen

firefox-printscreen-2


Google Chrome (53.0.2785.143 , 64 bits)
===============================

kytos-printscreen

kytos-printscreen-2

**Comment by diraol** [comment]
We cannot remove mapbox attributions because of the TOS.

> You must provide attribution, including an "Improve this map" link on all maps as outlined in our 
 documentation. You understand that a subscription to a Mapbox plan that offers "branding-free maps" does not eliminate your responsibility to provide attribution.

[TOS](https://www.mapbox.com/tos/#[YmpaYuta])
and [How Attribution Works](https://www.mapbox.com/help/how-attribution-works).

**Comment by macartur** [comment]
I don't know where I can put this MapBox attributions. Do you have some idea?

File: kytos/web-ui/static/js/kytos-background-map.js [hunk 1]
```diff
@@ -32,7 +32,9 @@ function draw_background_map(callback=undefined) {
           scrollWheelZoom: true,
           maxZoom: 20,
           minZoom: 2,
-          loadingControl: true
+          loadingControl: true,
+          attributionControl: false,
```
Comment (valessiobrito) [inline kytos/web-ui/static/js/kytos-background-map.js]: @macartur change to 'true' here?!

**Comment by macartur** [comment]
Google Chrome (53.0.2785.143 , 64 bits)
===============================

kytos-1

kytos-2

Firefox (54.0, 64bits)
================

kytos-firefox

**Review by diraol — APPROVED** [review]

**Review by cemsbr — APPROVED** [review]
There's no Mapbox in the bottom-left corner of Firefox, but this is good enough for me.

**Comment by diraol** [comment]
@cemsbr this is because firefox is rendering MapBox using 'mapboxlib.js' (or something like that), while chrome is using 'mapbox-gl'. The first one just present map tiles (images), while the latter renders the map with WebGL. So, they behave differently 'by origin'. Anyway, in both cases we do have the necessary Mapbox reference. =)

**Pull request merged**